sqflite/lib/src/sentry_database.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:meta/meta.dart';
+import 'package:sentry/sentry.dart';
+import 'package:sqflite/sqflite.dart';
+
+import 'sentry_database_executor.dart';
+import 'sentry_sqflite_transaction.dart';
+import 'version.dart';
+
+/// A [Database] wrapper that adds Sentry support.
+///
+/// ```dart
+/// import 'package:sqflite/sqflite.dart';
+/// import 'package:sentry_sqflite/sentry_sqflite.dart';
+///
+/// final database = await openDatabase('path/to/db');
+/// final sentryDatabase = SentryDatabase(database);
+/// ```
+class SentryDatabase extends SentryDatabaseExecutor implements Database {
+  final Database _database;
+  final Hub _hub;
+
+  /// ```dart
+  /// import 'package:sqflite/sqflite.dart';
+  /// import 'package:sentry_sqflite/sentry_sqflite.dart';
+  ///
+  /// final database = await openDatabase('path/to/db');
+  /// final sentryDatabase = SentryDatabase(database);
+  /// ```
+  SentryDatabase(
+    this._database, {
+    @internal Hub? hub,
+  })  : _hub = hub ?? HubAdapter(),
+        super(_database, hub: hub) {
+    // ignore: invalid_use_of_internal_member
+    final options = _hub.options;
+    options.sdk.addIntegration('SentrySqfliteTracing');
+    options.sdk.addPackage(packageName, sdkVersion);
+  }
+
+  @override
+  Future<void> close() {
+    Future<void> future() async {
+      final currentSpan = _hub.getSpan();
+      final span = currentSpan?.startChild(
+        'db',
+        description: 'CLOSE',
```

Comment:
l: I think is a good approach to use constants, easier to update later if necessary.